pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.black]
 line-length = 100
-target_version = ['py37', 'py38', 'py39']
+target_version = ['py38', 'py39']
```

Comment:
Should we also add `py310` here?

Reply:
> Should we also add `py310` here?

Yes, and py311 as well.

Reply:
Added both of these, and updated the black formatter because the old version didn't support 311